Error installing directus 9.12.0

### Describe the Bug

Fatal error when trying to install latest version of Directus (9.12.0).

### To Reproduce

Open cmd and try to install directus locally by typing:

`npm init directus-project example-project`

### Errors Shown

```
PS D:\dev> npm init directus-project fiscobra
node:internal/modules/cjs/loader:936
  throw err;
  ^

Error: Cannot find module 'ioredis'
Require stack:
- D:\dev\fiscobra\node_modules\directus\dist\messenger.js
- D:\dev\fiscobra\node_modules\directus\dist\services\flows.js
- D:\dev\fiscobra\node_modules\directus\dist\services\index.js
- D:\dev\fiscobra\node_modules\directus\dist\middleware\respond.js
- D:\dev\fiscobra\node_modules\directus\dist\controllers\activity.js
- D:\dev\fiscobra\node_modules\directus\dist\app.js
- D:\dev\fiscobra\node_modules\directus\dist\server.js
- D:\dev\fiscobra\node_modules\directus\dist\cli\index.js
- D:\dev\fiscobra\node_modules\directus\dist\cli\run.js
- D:\dev\fiscobra\node_modules\directus\cli.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (D:\dev\fiscobra\node_modules\directus\dist\messenger.js:7:35)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    'D:\\dev\\fiscobra\\node_modules\\directus\\dist\\messenger.js',
    'D:\\dev\\fiscobra\\node_modules\\directus\\dist\\services\\flows.js',
    'D:\\dev\\fiscobra\\node_modules\\directus\\dist\\services\\index.js',
    'D:\\dev\\fiscobra\\node_modules\\directus\\dist\\middleware\\respond.js',
    'D:\\dev\\fiscobra\\node_modules\\directus\\dist\\controllers\\activity.js',
    'D:\\dev\\fiscobra\\node_modules\\directus\\dist\\app.js',
    'D:\\dev\\fiscobra\\node_modules\\directus\\dist\\server.js',
    'D:\\dev\\fiscobra\\node_modules\\directus\\dist\\cli\\index.js',
    'D:\\dev\\fiscobra\\node_modules\\directus\\dist\\cli\\run.js',
    'D:\\dev\\fiscobra\\node_modules\\directus\\cli.js'
  ]
}
✖ Installing Directus
Error: undefined
npm ERR! code 1
npm ERR! path D:\dev
npm ERR! command failed
npm ERR! command C:\Windows\system32\cmd.exe /d /s /c create-directus-project "fiscobra"

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\lucas\AppData\Local\npm-cache\_logs\2022-06-03T20_32_27_512Z-debug-0.log
```

### What version of Directus are you using?

9.12.0

### What version of Node.js are you using?

16.14.2

### What database are you using?

Postgres 14

### What browser are you using?

Chrome

### How are you deploying Directus?

Locally

--- a/api/package.json
+++ b/api/package.json
@@ -119,6 +119,7 @@
 		"graphql-compose": "^9.0.1",
 		"helmet": "^4.6.0",
 		"inquirer": "^8.1.1",
+		"ioredis": "^4.27.6",
 		"joi": "^17.3.0",
 		"js-yaml": "^4.1.0",
 		"js2xmlparser": "^4.0.1",
@@ -164,7 +165,6 @@
 	},
 	"optionalDependencies": {
 		"@keyv/redis": "^2.1.2",
-		"ioredis": "^4.27.6",
 		"keyv-memcache": "^1.2.5",
 		"memcached": "^2.2.2",
 		"mysql": "^2.18.1",

